modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -8,6 +8,8 @@
 
 If you have not enabled autoscaling for your machine pool, you can manually scale the number of compute (also known as worker) nodes in the pool to meet your deployment needs.
 
+Scaling is performed within each machine pool separately.
```

Comment:
Please make this active. If you mean that the user must scale each pool, perhaps "You must scale each machine pool separately."